### PR TITLE
feat: add description text in expert transactions

### PR DESCRIPTION
**Pull request by 0xArdy**

### CODE REVIEW

File: src/components/BaseModal.tsx [hunk 1]
```diff
@@ -90,6 +90,7 @@ export const SModal = styled(Box)(() => {
     '@media (max-width: 600px)': {
       minWidth: '100%',
       margin: '0rem 1.6rem',
+      padding: '1.6rem 2rem 3.2rem',
```
Comment (turtlemoji) [inline src/components/BaseModal.tsx:93]: missing last one as 0

File: src/containers/Footer.tsx [hunk 2]
```diff
@@ -28,12 +28,21 @@ export const Footer = () => {
...
   margin-top: auto;
   width: 100%;
+
+  @media (max-width: 600px) {
+    display: flex;
+    margin-top: 16rem;
+    flex-direction: column;
+    align-items: start;
+    justify-content: center;
+    padding: 2rem 1.6rem 8rem;
```
Comment (turtlemoji) [inline src/containers/Footer.tsx:44]: idem?

**Review by turtlemoji — APPROVED** [review]
pre-approved with comments!

File: src/components/BaseModal.tsx (hunk 1 above)
Comment (0xArdy) [reply src/components/BaseModal.tsx:93]: i dont know if you refer to an old spec but when we use 3 values they are for: `top left+right bottom`.
https://developer.mozilla.org/en-US/docs/Web/CSS/padding

File: src/containers/Footer.tsx (hunk 2 above)
Comment (0xArdy) [reply src/containers/Footer.tsx:44]: answered here https://github.com/defi-wonderland/superchain-bridge-ui/pull/38#discussion_r1526169948

**Pull request merged**